From 33ae54fb27c620d05fb8b3812378395708817d57 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=E9=BB=84=E6=B6=9B?= <huangtao@rock-chips.com>
Date: Mon, 16 Jul 2012 10:23:39 +0800
Subject: [PATCH] Revert "rk30: clock: disable pd_vio gating function before
 rga, lcdc, cif support properly"

This reverts commit 2186d7743ad119ca8b5eea702970463fe4aba3df.
---
 arch/arm/mach-rk30/clock_data.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/mach-rk30/clock_data.c b/arch/arm/mach-rk30/clock_data.c
index 3ef0cdfad4f2..7579235ebabd 100644
--- a/arch/arm/mach-rk30/clock_data.c
+++ b/arch/arm/mach-rk30/clock_data.c
@@ -2458,7 +2458,7 @@ static int pd_display_mode(struct clk *clk, int on)
 static struct clk pd_display = {
 	.name	= "pd_display",
 	.flags  = IS_PD,
-//	.mode	= pd_display_mode,
+	.mode	= pd_display_mode,
 	.gate_idx	= PD_VIO,
 };
 
-- 
2.35.3

